src/components/store/reducer.ts

```diff
@@ -0,0 +1,30 @@
+import { EditorState } from '../../../types/store/editorState';
+import { Action } from '../../../types/store/action';
+import * as _ from '../utils';
+import { BlockMutationType } from '../../../types/events/block/mutation-type';
+
+/**
+ * Reducer function for Editor.js state
+ * This function applies the passed action to the current state and returns the new state
+ *
+ * @param state - previous state to apply action
+ * @param action - information about the action in the previous state
+ */
+const reducer = (state: EditorState, action: Action): EditorState => {
```

Comment:
why it is the anonymous function? 

```suggestion
export default function reducer(state: EditorState, action: Action): EditorState {
```